admittance_controller/src/admittance_controller.cpp

```diff
@@ -265,6 +283,20 @@ controller_interface::CallbackReturn AdmittanceController::on_configure(
   input_joint_command_subscriber_ =
     get_node()->create_subscription<trajectory_msgs::msg::JointTrajectoryPoint>(
       "~/joint_references", rclcpp::SystemDefaultsQoS(), joint_command_callback);
+
+  input_wrench_command_subscriber_ =
+    get_node()->create_subscription<geometry_msgs::msg::WrenchStamped>(
+      "~/wrench_reference", rclcpp::SystemDefaultsQoS(),
+      [&](const geometry_msgs::msg::WrenchStamped & msg)
+      {
+        if (msg.header.frame_id != admittance_->parameters_.ft_sensor.frame.id && !msg.header.frame_id.empty())
+        {
+          RCLCPP_ERROR_STREAM(
```

Comment:
Why don't we transform the wrench if it is in another frame to the sensor frame?

Reply:
I had some very weird results in the past when transforming wrenches with TF (e.g. non zero torques afterwards). Additionally I don't think we have a tf buffer/listener in the controller at the moment and why add this additional overhead?